### PR TITLE
support 1dcnn distilling

### PR DESCRIPTION
**Pull request by priscilla-pan**
#24

### CODE REVIEW

File: doc/CNN-1D-tiny-Distillation.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+# Tiny 1D-CNN Knowledge Distillation
+
+The following uses 1D-CNN on the 12 classes session all dataset as teacher model to illustrate how to use the model  
+optimizer to improve the preformance of tiny 1D-CNN by knowledge distillation.
```
Comment (ChengcanWang-zte) [inline doc/CNN-1D-tiny-Distillation.md]: performance

File: doc/CNN-1D-tiny-Distillation.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+
+The 1D-CNN model is from Wang's paper[Wang, W.; Zhu, M.; Wang, J.; Zeng, X.; Yang, Z. End-to-end encrypted traffic  
+classification with one-dimensional convolution neural networks.] The tiny 1D-CNN model is a slim version of the
+1D-CNN model mentioned before. Using 1D-CNN model as the teacher to ditstill tiny 1D-CNN model, performance can be
```
Comment (ChengcanWang-zte) [inline doc/CNN-1D-tiny-Distillation.md]: to distill

File: doc/CNN-1D-tiny-Distillation.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+| cnn1d_tiny+ distill | 93.28%   | 134988 | 546K|
+
+## 1 Create custom dataset
+
+Using [ISCX dataset](https://www.unb.ca/cic/datasets/vpn.html), you can get the processed 12-classes-session-all dataset
+from [wang's github](https://github.com/echowei/DeepTraffic/blob/master/2.encrypted_traffic_classification/3.PerprocessResults/12class.zip).
+We name the dataset as iscx_session_all. In the iscx_session_all, there are 35501 training samples, the shape is
+(35501, 28, 28), 3945 testing samples.  
+
+Now that you have the dataset, you can implement your custom dateset by extending model_optimizer.prunner.dataset.
```
Comment (ChengcanWang-zte) [inline doc/CNN-1D-tiny-Distillation.md]: dateset to dataset

File: doc/CNN-1D-tiny-Distillation.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+python3 convert_softmax_model_to_logits.py
+```
+
+After execution, the default checkpoint file of logits model will be generated in examples/models_eval_ckpt/cnn1d/
+checkpoint-60-logits.h5
+
+## Create the distilling process and distill the cnn1d_tiny model
+
+Create the configuration file in the src/model_optimizer/pruner/scheduler/distill,like "cnn1d_tiny_0.3.yaml" where the
+distillation parameters is configured.
```
Comment (ChengcanWang-zte) [inline doc/CNN-1D-tiny-Distillation.md]: is to are
